storage/libradb/src/pruner/mod.rs

```diff
@@ -158,6 +158,7 @@ impl Worker {
             // in case `Command::Quit` is received (that's when we should quit.)
             match prune_state(
                 Arc::clone(&self.db),
+                self.progress.load(Ordering::Relaxed),
```

Comment:
Also `Relaxed` on line 170? Otherwise it is confusing to pair `Relaxed` with `Release`.

Reply:
@msmouse ?

Reply:
the `Acquire` and `Release` pair exists between the worker and main threads, this is in the same thread of the `Release` so I used Relaxed. But as I thought of it I realized you were right -- `Relaxed` is adequate for all situations here. set #485